### PR TITLE
OpenShift section is added to installation.md

### PR DESCRIPTION
**Pull request by mareklibra**
Instructions for adding SCCs are listed.

### CODE REVIEW

File: installation.md [hunk 1]
```diff
@@ -76,6 +76,27 @@ Or to have convenience commands for:
...
+oc create serviceaccount -n kube-system kubevirt
+```
+
+and following [SCCs](https://docs.openshift.com/container-platform/3.7/admin_guide/manage_scc.html) needs to be added:
+```
+oc adm policy add-scc-to-user privileged system:serviceaccount:kube-system:kubevirt-privileged
+oc adm policy add-scc-to-user privileged system:serviceaccount:kube-system:kubevirt-controller
+```
+
+Please note, the `kubevirt` service account is created even within `kubevirt.yaml` but creating it explicitly allows adding SCCs before `kubevirt.yaml` is deployed.
```
Comment (fabiand) [inline installation.md]: How does this make a difference?

File: installation.md [hunk 2]
```diff
@@ -76,6 +76,27 @@ Or to have convenience commands for:
...
+
+### Kubevirt 0.2.0
```
Comment (fabiand) [inline installation.md]: Please move this to the beginnig of the section so readers read it before deployment
,a lso maybe use bold \*\*NOTE\*\*:

File: installation.md (hunk 1 above)
Comment (mareklibra) [reply installation.md]: oh, right. No difference, declarative approach.

File: installation.md (hunk 2 above)
Comment (mareklibra) [reply installation.md]: done

**Comment by fabiand** [comment]
Thanks!

**Pull request merged**